runserver.py

```diff
@@ -1,17 +1,50 @@
-#!/usr/bin/env python
+#!/usr/bin/env python2
 
 # These two lines are needed to run on EL6
-__requires__ = ['SQLAlchemy >= 0.7', 'jinja2 >= 2.4']
+__requires__ = ['SQLAlchemy >= 0.8', 'jinja2 >= 2.4']
 import pkg_resources
 
+import argparse
 import sys
-from werkzeug.contrib.profiler import ProfilerMiddleware
+import os
 
-from mirrormanager2.app import APP
-APP.debug = True
 
-if '--profile' in sys.argv:
+parser = argparse.ArgumentParser(
+    description='Run the mirrormanager2 app')
+parser.add_argument(
+    '--config', '-c', dest='config',
+    help='Configuration file to use.')
+parser.add_argument(
+    '--debug', dest='debug', action='store_true',
+    default=False,
+    help='Expand the level of data returned.')
+parser.add_argument(
+    '--profile', dest='profile', action='store_true',
+    default=False,
+    help='Profile the web application.')
+parser.add_argument(
+    '--port', '-p', default=5000,
+    help='Port for the flask application.')
+parser.add_argument(
+    '--host', default="127.0.0.1",
+    help='Hostname to listen on. When set to 0.0.0.0 the server is available \
+    externally. Defaults to 127.0.0.1 making the it only visable on localhost')
+
+args = parser.parse_args()
+
+if args.profile:
+    from werkzeug.contrib.profiler import ProfilerMiddleware
     APP.config['PROFILE'] = True
     APP.wsgi_app = ProfilerMiddleware(APP.wsgi_app, restrictions=[30])
 
```

Comment:
APP is imported later. This will probably fail. Although, I don't know what and how the profile is used.